tests/test_fsclient.py

```diff
@@ -143,11 +144,15 @@ def test_insert_one_bad(rc):
 
     client.load_database(pydr_rc['databases'][0])
 
-    with pytest.raises(KeyError):
+    with pytest.raises(KeyError) as excinfo:
         client.insert_one(dbname, collname, test_insert_cif_bad[0])
-    with pytest.raises(TypeError):
-        client.insert_one(dbname, collname, test_insert_cif_bad[1])
+    assert '_id' in str(excinfo.value)
+    print(excinfo)
```

Comment:
Also, how do we return info back to the user in pytest? Is print() enough? I couldn't find it on pytest docs...

Reply:
then we need logic in the function itself so it raises an exception with the right error message.